src/utils/parser/tests/getSymbols.spec.js

```diff
@@ -53,4 +53,9 @@ describe("Parser.getSymbols", () => {
     const symbols = formatSymbols(getSource("parseScriptTags", "html"));
     expect(symbols).toMatchSnapshot();
   });
+
+  it("component", () => {
+    const symbols = formatSymbols(getSource("component"));
+    expect(symbols).toMatchSnapshot();
+  });
```

Comment:
lets update `formatSymbols` to show the class name when it exists, this will get us some additional test coverage.